.dotnet/src/Custom/Images/GeneratedImage.cs

```diff
@@ -15,13 +16,13 @@ public class GeneratedImage
     /// This property is mutually exclusive with <see cref="ImageUri"/> and will be <c>null</c> when the other
     /// is present.
     /// </remarks>
-    public BinaryData ImageBytes { get; }
+    public Stream Image { get; }
```

Comment:
I'll take this out of this PR too, to focus on inputs, but if we decide to use protocol methods as the way to get Stream outputs for large binary content (instead of BinaryData), this would be another area affected -- we should minimally look at what the corresponding protocol method looks like and how complicated a sample would be to get the output image content as a Stream.